Shared/km.lproj/Menu.strings

```diff
@@ -1,69 +1,18 @@
-/* Label for the button, displayed in the menu, used to create a bookmark for the current website. */
-"Menu.AddBookmarkAction.Title" = "ចំណាំ​ទំព័រ​នេះ";
-
-/* Label for the button, displayed in the menu, used to add a page to the reading list. */
-"Menu.AddToReadingList.Title" = "បញ្ចូល​​ទៅ​បញ្ជី​អាន";
-
 /* Label for the button, displayed in the menu, used to close all tabs currently open. */
 "Menu.CloseAllTabsAction.Title" = "បិទ​ផ្ទាំង​ទាំងអស់​";
 
 /* Label for the button, displayed in the menu, used to copy the page url to the clipboard. */
 "Menu.CopyAddress.Title" = "ចម្លងអាសយដ្ឋាន";
 
-/* Label for the button, displayed in the menu, used to open the toolbar to search for text within the current page. */
```

Comment:
@boek same here we have a lot of strings gone (infact all ha) so just making sure thats okay